DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Speckle.Connectors.RevitShared.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.Build.NoTargets/3.3.0">
```

Comment:
This is what makes it shared, I guess?

Reply:
This csproj isn't used at all so we don't want msbuild to do anything but can be seen in the sln